library/src/main/java/com/novoda/downloadmanager/lib/NotificationsCreatedListener.java

```diff
@@ -0,0 +1,33 @@
+package com.novoda.downloadmanager.lib;
+
+import android.app.Notification;
+import android.app.Service;
+import android.support.v4.util.SimpleArrayMap;
+
+import com.novoda.downloadmanager.notifications.NotificationTag;
+
+import static com.novoda.downloadmanager.notifications.DownloadNotifier.TYPE_ACTIVE;
+
+public class NotificationsCreatedListener {
```

Comment:
This can't be moved unless the packages are collapsed. It's created in `lib` and used in `notifications`. I'd rather the constructor be package the methods accessible because at least the clients can't create it to use it.